src/Admin/CategoryAdmin.php

```diff
@@ -63,6 +63,10 @@ public function getFormBuilder()
      */
     protected function configureFormFields(FormMapper $formMapper)
     {
+        $categoryManager = $this->getConfigurationPool()
+            ->getContainer()
+            ->get('sonata.classification.manager.category');
```

Comment:
This should be obtained with constructor injection.

Reply:
@gremo what about this?

Reply:
@greg0ire no problem I can change it, I was talking about the entire PR... what do you think?

Reply:
Uuuuh it looks good I guess